rootfs/etc/profile.d/prompt.sh

```diff
@@ -91,6 +91,15 @@ function geodesic_prompt() {
 		;;
 	esac
 
+	local level_prompt
+	case $SHLVL in
+	1) level_prompt='.' ;;
+	2) level_prompt=':' ;;
+	3) level_prompt='⋮' ;; # vertical elipsis \u22ee from Mathematical Symbols
+	*) level_prompt="$SHLVL" ;;
```

Comment:
Maybe? Haven't seen what it looks like.
```suggestion
	*) level_prompt="($SHLVL)" ;;
```